uw-frame-components/portal/misc/partials/circle-button.html

```diff
@@ -1,5 +1,6 @@
 <md-button ng-href="{{ cbDisabled ? '#' : href}}"
            target="{{cbDisabled ? '' : target}}"
+           rel='noopener noreferrer'
```

Comment:
Are there negative consequences of *always* using `noopener noreferrer` even when `target` is *not* `_blank`?